.github/workflows/build.yml

```diff
@@ -15,20 +15,27 @@ jobs:
   host_builds:
     strategy:
       matrix:
-        os: [macos-latest, windows-2022]
+        include:
+          - os: macos-latest
+            arch: ''
```

Comment:
Is `arch` required in this case? Looks like empty string is the default value, so there's no need to manually specify, suggest removing this line and line 22.